src/pvrclient-nextpvr.cpp

```diff
@@ -1734,13 +1752,18 @@ PVR_ERROR cPVRClientNextPVR::UpdateTimer(const PVR_TIMER &timerinfo)
 
 /************************************************************/
 /** Live stream handling */
-
 bool cPVRClientNextPVR::OpenLiveStream(const PVR_CHANNEL &channelinfo)
 {
+  if (channelinfo.iUniqueId == m_iLiveStreamUID)
+  {
+    XBMC->Log(LOG_NOTICE, "New channel uid equal to the already streaming channel. Skip re-tune.");
+    return true;
```

Comment:
Why this change? Is this a theoretical case or something you experienced?

Reply:
I was thinking the same thing but didn't bother to comment

Reply:
It's theoretical, given the check a few lines below that there's already a stream open, it seemed prudent._